src/ConnectedMode/UI/ManageBinding/ManageBindingViewModel.cs

```diff
@@ -141,12 +141,10 @@ public async Task InitializeDataAsync()
             UiResources.FetchingBindingStatusFailedText) { AfterProgressUpdated = OnProgressUpdated };
         await ProgressReporter.ExecuteTaskWithProgressAsync(displayBindStatus);
 
-        if (!IsCurrentProjectBound)
-        {
-            var detectSharedBinding = new TaskToPerformParams<AdapterResponse>(CheckForSharedBindingAsync, UiResources.CheckingForSharedBindingText,
-                UiResources.CheckingForSharedBindingFailedText) { AfterProgressUpdated = OnProgressUpdated };
-            await ProgressReporter.ExecuteTaskWithProgressAsync(detectSharedBinding);
-        }
+        var detectSharedBinding = new TaskToPerformParams<AdapterResponse>(CheckForSharedBindingAsync, UiResources.CheckingForSharedBindingText,
+                UiResources.CheckingForSharedBindingFailedText)
+            { AfterProgressUpdated = OnProgressUpdated };
+        await ProgressReporter.ExecuteTaskWithProgressAsync(detectSharedBinding);
```

Comment:
The issue seems to be fixed by https://github.com/SonarSource/sonarlint-visualstudio/pull/5906 This change can be reverted.

Reply:
My bad, I started with an unbound project which is not the same.